.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: php
 
+# We do Behat testing here, lots of sudo going on
+sudo: required
+
 # run tests on php 5.5 only and let unit / integration tests deal with php differences
 php:
   - 5.5
 
 # list of paths/bundle to execute
 env:
-    # run setup wizard to install Demo+Content and then test demo with content
   - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
-    # test Legacy
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="legacyAdmin" TEST="full"
-    # test REST
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="rest" TEST="fullJson"
-  - INSTALL="demoCleanNonUniqueDB" PROFILE="rest" TEST="fullXml"
+  - PROFILE="demo" TEST="clean"
+  - PROFILE="rest" TEST="fullJson"
+  - PROFILE="rest" TEST="fullXml"
```

Comment:
yes the env part, at least I think there is some logic around the use of 'INSTALL' for legacy, but it might be gone.